File: javascript/이상한 문자 만들기.js
const solution = (s) => {
    let answer = '';
    let sList = s.split(' ');
    for(let word of sList) {
        for(let i = 0; i < word.length; i++) {
            i % 2 == 1 ? answer += word[i].toLowerCase() : answer += word[i].toUpperCase()
        }
        answer += ' '
    }
    return answer.slice(0 , -1);
}

File: java/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] array = s.split("");
        int idx = 0;
        
        for(String str : array) {
            idx = str.contains(" ") ? 0 : idx + 1;
            answer += idx % 2 == 0 ? str.toLowerCase() : str.toUpperCase();
        }
        return answer;
    }
}